capstone/src/main/webapp/index.html.orig

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>INSERT TITLE HERE</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBtX12dX0EUC2BdYADIIxZ8IRFvlz9VUfI"></script>
+    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Oswald&family=Raleway&display=swap" rel="stylesheet">
+  </head>
+<<<<<<< HEAD
```

Comment:
I see that the merge conflict indicators are within this file itself. May have to clean out the index.html